.travis.yml

```diff
@@ -72,7 +72,7 @@ matrix:
       addons: &gcc6
         apt:
           sources: ['ubuntu-toolchain-r-test']
-          packages: ['g++-6', 'libbz2-dev', 'libstxxl-dev', 'libstxxl1', 'libxml2-dev', 'libzip-dev', 'lua5.1', 'liblua5.1-0-dev', 'libtbb-dev', 'libgdal-dev', 'libluabind-dev', 'libboost-all-dev']
+          packages: ['g++-6', 'libbz2-dev', 'libxml2-dev', 'libzip-dev', 'lua5.1', 'liblua5.1-0-dev', 'libtbb-dev', 'libgdal-dev', 'libboost-all-dev']
```

Comment:
I'm curious why Travis passes with this, since we still have a dependency on STXXL [here](https://github.com/Project-OSRM/osrm-backend/blob/master/src/extractor/edge_based_graph_factory.cpp#L337) see https://github.com/Project-OSRM/osrm-backend/blob/master/include/extractor/turn_data_container.hpp#L117 with https://github.com/Project-OSRM/osrm-backend/blob/master/include/util/vector_view.hpp#L212-L216